MonMaraicher/Model/FarmerPlace.swift

```diff
@@ -13,27 +13,97 @@ struct FarmerPlace: Identifiable, Hashable {
     let location: Location
     let imageNames: [String]
 
-    static let all = [FarmerPlace(name: "Chez William", location: FarmerPlace.applePark1, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"]),
-                      FarmerPlace(name: "Maraîchers Bio de Grabels", location: FarmerPlace.applePark2, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"]),
-                      FarmerPlace(name: "Fred producteur", location: FarmerPlace.applePark3, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"]),
-                      FarmerPlace(name: "Mon Maraîcher", location: FarmerPlace.applePark4, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"]),
-                      FarmerPlace(name: "La ferme du Chot", location: FarmerPlace.applePark5, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"]),
-                      FarmerPlace(name: "Chez William", location: FarmerPlace.chezWilliam, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"]),
-                      FarmerPlace(name: "Maraîchers Bio de Grabels", location: FarmerPlace.bio, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"]),
-                      FarmerPlace(name: "Fred producteur", location: FarmerPlace.fred, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"]),
-                      FarmerPlace(name: "Mon Maraîcher", location: FarmerPlace.maraicher, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"]),
-                      FarmerPlace(name: "La ferme du Chot", location: FarmerPlace.chot, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"])
+    static let all = [FarmerPlace(name: "chez william", location: FarmerPlace.applePark1, imageNames: ["farmer1", "farmer2", "farmer3", "farmer4", "farmer5", "farmer6"]),
```

Comment:
The file is becoming huge.. I would move all static inside another file to avoid mixing your data with the model declaration. It's up to you ;)